### PR TITLE
Hum 39 click on human app logo should redirect to default page

### PR DESCRIPTION
**Pull request by KacperKoza343**
## Description

HUM-39

### CODE REVIEW

File: packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx [hunk 1]
```diff
@@ -72,7 +74,17 @@ export function DrawerNavigation({
...
       >
         {!isMobile && (
-          <Stack alignItems="flex-start" sx={{ paddingLeft: '60px' }}>
+          <Stack
+            alignItems="flex-start"
+            sx={{ paddingLeft: '60px', cursor: 'pointer' }}
+            onClick={() => {
+              handleMainNavIconClick();
+              // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+              if (isMobile) {
```
Comment (dnechay) [inline packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx:83]: This component is rendered only for desktop (`!isMobile &&` above), do we need this code block at all?

File: packages/apps/human-app/frontend/src/pages/worker/jobs/components/available-jobs/mobile/available-jobs-drawer-mobile.tsx [hunk 2]
```diff
@@ -58,7 +60,15 @@ export function AvailableJobsDrawerMobile({
             zIndex: '999999',
           }}
         >
-          <HumanLogoIcon />
+          <Stack
+            sx={{ cursor: 'pointer' }}
+            onClick={() => {
+              handleMainNavIconClick();
+              // setIsMobileFilterDrawerOpen(false);
```
Comment (dnechay) [inline packages/apps/human-app/frontend/src/pages/worker/jobs/components/available-jobs/mobile/available-jobs-drawer-mobile.tsx:67]: Do we need this comment?

**Review by dnechay — COMMENTED** [review]
Not sure why we have multiple components for mobile logo, but it's not related to this PR.

Main change lgtm, some minor cleanups and should be good to go

**Comment by KacperKoza343** [comment]
> Not sure why we have multiple components for mobile logo, but it's not related to this PR.
> 
> Main change lgtm, some minor cleanups and should be good to go
cleanups done

File: packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx [hunk 3]
```diff
@@ -72,7 +74,13 @@ export function DrawerNavigation({
         variant="persistent"
       >
         {!isMobile && (
-          <Stack alignItems="flex-start" sx={{ paddingLeft: '26px' }}>
+          <Stack
+            alignItems="flex-start"
+            sx={{ paddingLeft: '60px', cursor: 'pointer' }}
```
Comment (dnechay) [inline packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx:79]: Should it be 60px?

Looks a bit too much in comparison to design
Screenshot 2024-10-25 at 18 47 08
Screenshot 2024-10-25 at 18 47 40

**Review by dnechay — COMMENTED** [review]
lgtm in general

One question about padding before approve

File: packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx (hunk 3 above)
Comment (dnechay) [reply packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx]: @KacperKoza343 ☝🏼

**Review by dnechay — APPROVED** [review]

**Pull request merged**